src/ontology/reports/doid_exclusion_reasons.robot.template.tsv

```diff
@@ -530,6 +530,9 @@ DOID:0110638	MONDO:excludeNonDisease
 DOID:0110649	MONDO:excludeNonDisease
 DOID:0110748	MONDO:excludeNonDisease
 DOID:0110762	MONDO:excludeNonDisease
+DOID:0111201	MONDO:excludeNonDisease
+DOID:0111204	MONDO:excludeNonDisease
+DOID:0111208	MONDO:excludeNonDisease
```

Comment:
@matentzn I don't know why these are in the exclusion list: there are currently as equivalent in Mondo